code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -139,16 +139,26 @@
 													//WHO THE FUCK THOUGHT THAT WAS A GOOD FUCKING IDEA!?!?
 
 		if("howl", "howls")
-			var/M = handle_emote_param(param) //Check to see if the param is valid (mob with the param name is in view).
-			message = "<B>[src]</B> howls[M ? " at [M]" : ""]!"
-			playsound(loc, 'sound/goonstation/voice/howl.ogg', 100, 1, 10, frequency = get_age_pitch())
-			m_type = 2
+			if(isvulpkanin(src)) 				  //Need this again for some reason, although it's already checked earlier
+				var/M = handle_emote_param(param) //Check to see if the param is valid (mob with the param name is in view).
+				if(miming)
+					message = "<B>[src]</B> acts out a howl[M ? " at [M]" : ""]!"
+					m_type = 1
```

Comment:
you could take this opportunity to change the `m_type` values into defines, that way there aren't magic numbers all over the place.

Reply:
Would be great if a define was made for these yes. 1 or 2 doesn't mean anything when you read it.

Reply:
True, but that'd mean editing every emote in existence, which feels a bit outside the scope of this PR, and once you read the emote code, the magic numbers make a bit more sense.

Reply:
Use search and replace and you've got it done in five minutes. Excuses.